### ISSUE TITLE
[Data Issue]: The country comparison of CO2 emissions is useless due to the incomparable values per technology and zone

### ISSUE THREAD

**Issue by lewie** [electricitymaps/electricitymaps-contrib#6743]
### When did this happen?

global

### What zones are affected?

All

### What is the problem?

The country comparison of CO2 emissions is useless due to the incomprehensible values per technology and country.

https://twitter.com/HannoKlausmeier/status/1789912185658921158
This ist a copy of the analysis of your carbon intensity values per country from Hanno Klausmeier.

The rather clear conclusion, see below, allows only one conclusion. Electricitymaps cannot be recommended and is completely unsuitable as a country comparison, especially when it comes to CO2 emissions. I very much doubt that the lack of comparability of such important values is the aim and purpose of this project.

I am not the only one who would be very pleased to receive a statement and explanation from the people responsible for the values.
Thank you very much

> Electricity maps: I took a deeper look at electricity maps. 
> What highlighted my attention? 
> It is a Danish company but with French decision makers coming from ecole centrale Paris. Those people are typically very nuclear minded.
> I also took a look at their data. Interesting is that the carbon intensity for the same "ingredient" varies from country to country. You may compare gCO2/kwh gas in Germany, Spain and the UK. The numbers are different. Why? I can hardly see any scientific reason. 
> It seems rather pretty obvious that the authors are trying to push one electricity source over an other. 
> Another point. Nuclear is set with the absolute minimum 5 gCO2/kwh. I am not saying it is wrong however the literature goes even up to 116 gCO2/kwh. On the other hand Germany and Poland is always "punished" with the worst data electricity maps could get hold on. For Germany electricity maps takes the worst data that they could get hold on just using lignite data. Hard coal has however a different carbon intensity!
> Why not differentiate lignite from hard coal and take an average value? 
> My conclusion is tough and hard: Electricity maps is not a reliable source of information and is even manipulative in its representation. I used to take a look at their data in the past but I will stop doing so. It is obvious to my that they are just pushing the nuclear agenda. I am not saying that those numbers are THE RIGHT ones but is shows that electricity maps is always taking THE BEST numbers for nuclear and always the worst for Germany and their Energiewende.
> 
> Here some additional remarks from Holger Mitterwald about their bugtracker:
> 
> It is striking that the absolute lower values for CO2 are used for nuclear power and very high values for coal, PV and wind. Whereas previously the values were taken uniformly from the IPCC reports, now they are wildly compiled from different papers according to taste.
> 
> In some cases with completely different calculation methods. Lignite and hard coal are lumped together. For German power plants, very precise CO2 values are available from a study that takes into account upstream and downstream emissions. For other countries, very standard values are used that do not take into account the upstream and downstream stages and are only based on estimates. But that's not all. Despite precise values in Germany, an average is calculated for all power plants of a given type, which also includes “dirty” power plants that have been shut down for a long time. In the case of gas, around 65% (!) of gas-fired power plant output is not taken into account at all. It is mainly small, old, inefficient power plants with a poor CO2 balance that are taken into account. Coincidences? Hardly imaginable. Someone knew exactly what they were doing.
> 
> How do we know that? Electricity-Maps is open-source. The source code is available, the bug tracker is available, the calculation methods of Electricity-Maps are described in great detail. You just have to make the effort to delve a little deeper.
> 
> Conclusion: Since Electricity-Maps has “improved” its calculations, the values are no longer comparable between countries because the sources of the initial values are based on completely different, non-comparable assumptions.

**Comment by VIKTORVAV99** [comment]
Hi!
We are aware and are discussing solutions to this internally but it will take some time before we have something actionable that can be used in the app.

**Comment by lewie** [comment]
Thanks for your quick reply.
I think it might be a nobrainer to use the official IPCC figures in parallel to the difficult to understand country-specific non-comparable CO2 figures (for example via a toggle switch)!
https://www.ipcc.ch/site/assets/uploads/2018/02/ipcc_wg3_ar5_chapter7.pdf
![grafik](https://github.com/electricitymaps/electricitymaps-contrib/assets/5707426/1ad606d5-879e-42d1-97ab-16561345a9c8)

**Comment by VIKTORVAV99** [comment]
> Thanks for your quick reply. I think it might be a nobrainer to use the official IPCC figures in parallel to the difficult to understand country-specific non-comparable CO2 figures (for example via a toggle switch)! https://www.ipcc.ch/site/assets/uploads/2018/02/ipcc_wg3_ar5_chapter7.pdf ![grafik](https://private-user-images.githubusercontent.com/5707426/329980094-1ad606d5-879e-42d1-97ab-16561345a9c8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU1OTM0NzUsIm5iZiI6MTcxNTU5MzE3NSwicGF0aCI6Ii81NzA3NDI2LzMyOTk4MDA5NC0xYWQ2MDZkNS04NzllLTQyZDEtOTdhYi0xNjU2MTM0NWE5YzgucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDUxMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA1MTNUMDkzOTM1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YzIwZGI3MDNhMTU2MjFhZGUwZTYyNDYzOTM5NGUyY2EyMTFjZGUxNjdiOTNjZjc1ZGNkNmI2MWU0ZTI3ZDliMSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.d37IlmaxVZdfPc-v8SXmQnryW3dJfgjN8YGNOkGJvBA)

Yes that is our thinking as well!
The current issue is how we integrate it into our data pipeline to process both the regional emissions and the IPCC emissions as the regional is still needed for our API consumers and carbon accounting data.

And then how we integrate both in the app because we still think the regional emissions have a lot of value even to the app users.

Hopefully we can work on this soon!

**Comment by Graefer** [comment]
What are you aware of? Have you read the posting completely? You are accused of intentionally using wrong regional data and of following a pro-nuclear agenda. Please clarify!

> Hi! We are aware and are discussing solutions to this internally but it will take some time before we have something actionable that can be used in the app.

**Comment by VIKTORVAV99** [comment]
> What are you aware of? Have you read the posting completely? You are accused of intentionally using wrong regional data and of following a pro-nuclear agenda. Please clarify!
> 
> > Hi! We are aware and are discussing solutions to this internally but it will take some time before we have something actionable that can be used in the app.

We are aware that you can't really compare the zones 1:1 like people seemingly expect to due to the regional emission facts. This is not because they are wrong but because there are regional variations due to construction methods, transport, infrastructure, and more that by the very nature of things can't be identical across the world. And in the case of solar even solar irradiance plays in as to how much co2eq is emitted per kWh as the production efficiency changes.

So this has nothing to do with nuclear but something that is true for all production sources. 

Our regional emission factors aims to be the most accurate representation of the actual grid emission of a zone but it does so at the cost of comparability between zones. Something we would like to improve if it's possible.

But we won't do so unless we can find an understandable and clear way to communicate that the values are for comparison purposes only and not meant to taken as hard truths. Something we are seeing people doing even for our own estimations even when they are clearly marked as just estimations.

**Comment by defipunk** [comment]
Your numbers are severely incorrect even within a zone. If you read the thread, to use the German example, you are using the worst possible values for coal and gas instead of the actual, also known and reported values

**Comment by lewie** [comment]
> Our regional emission factors aims to be the most accurate representation of the actual grid emission of a zone but it does so at the cost of comparability between zones. Something we would like to improve if it's possible.

I can well understand that. However, this strategy seems implausible if, for example, only the emissions during the operating phase of nuclear technology are used for all countries for nuclear plants, but the emissions with upstream chains and disposal are used for other energy technologies.

Take Finland, for example. A new nuclear power plant is operating there. Like all other nuclear plants, this is assumed to have (source: UNECE 2022, value: 5.13). Just like the plants in France that have long since been written off. How can that be?
This extremely low value of 5.13 gr/kWh is extremely low even for the pure utilization phase and completely ignores all infrastructures outside the power plant, upstream chains, storage, dismantling and disposal.

I think that instead of giving values for the life cycle, it would be possible to give values for the utilization phase only. But then, for example, the CO2 emissions from photovoltaics would have to be given as a blanket figure of almost 0 gr/kWh!

What is the value of such a finely structured system of emission values if such blatant error values are used?

**Comment by Kezii** [comment]
This thread has been polluted by anti-nuclear militants and electricitymap developers should take this into account


> Take Finland, for example. A new nuclear power plant is operating there. Like all other nuclear plants, this is assumed to have (source: UNECE 2022, value: 5.13). Just like the plants in France that have long since been written off. How can that be?


France's nuclear fleet is calculated to be around 3.7gCO2eq/kWh, that is less than the UNECE average that you quoted, and that includes infrastructures outside the power plant, etc [1]. does that mean that electricity map is anti-nuclear now?


> This extremely low value of 5.13 gr/kWh is extremely low even for the pure utilization phase and completely ignores all infrastructures outside the power plant, upstream chains, storage, dismantling and disposal.

This is completely wrong, from the same document you quoted [2]
Their estimate is even conservative, as they don't count the lower emissions of using reprocessed fuel [3] (fiy france do reprocess fuel)




please stop lying and wasting people's time

[1] ![immagine](https://github.com/electricitymaps/electricitymaps-contrib/assets/3357750/79fbf543-2ae6-4fb1-a104-7667230bbabe)
[2] 
![immagine](https://github.com/electricitymaps/electricitymaps-contrib/assets/3357750/a26db798-a321-4b94-b56d-4640d99006f2)

[3] 
![immagine](https://github.com/electricitymaps/electricitymaps-contrib/assets/3357750/df50dc9b-4152-48c8-92c8-58004b24a723)

**Comment by postnormaltimes** [comment]
> This extremely low value of 5.13 gr/kWh is extremely low even for the pure utilization phase and completely ignores all infrastructures outside the power plant, upstream chains, storage, dismantling and disposal.

 This comment perfectly explains how you don't understand how a Life Cycle Analysis is conducted, the methodology used and inputs needed. It was expected, as the "outstanding analysis" (being an X thread...) you posted is riddled with errors and written by a known anti-nuclear account on that social media (his only intent with the post was to make it seem as if Electricty Maps favored nuclear and nuclear-powered countries...)

The only real issue raised is that countries emissions can't really be compared 1:1, but it's not because the data is wrong...study and come back

**Comment by tonypls** [comment]
Thank you for bringing your concerns to our attention. We appreciate the community's involvement in striving for accuracy and transparency in our data. However, the issue raised here seems to be based on misunderstandings or misinterpretations of the data and methodologies we use.

Our platform aims to provide the most accurate representation of the reality of electricity emissions, which inherently includes variations due to local factors. We maintain transparency about these differences, as detailed in the links below.

We recognize the complexities involved in comparing data across different regions due to these factors. However, our commitment remains to provide the most accurate representation of carbon intensities based on scientifically validated methodologies.

We are committed to neutrality (I'm from New Zealand and we've been nuclear free since the 1987) and scientific integrity. We encourage our users to delve into our open-source materials for a clearer understanding of how our data is compiled and the scientific basis behind it.

Given the extensive discussion and our ongoing efforts to incorporate community feedback into improving our data representation, we are closing this issue. We invite further discussion on specific technical proposals in new issues or updates to existing issues. For example on this newly created issue: #6749 

Thank you again for your engagement and support in improving our platform.

Please find more about our methodology: [here](https://www.electricitymaps.com/methodology#carbon-intensity-and-emission-factors) and our zone specific emission factors: [here](https://github.com/electricitymaps/electricitymaps-contrib/blob/master/EMISSION_FACTORS_SOURCES.md)
